### ISSUE TITLE
Error during start up

### ISSUE THREAD

**Issue by Dantalion** [omulet/react-native-radial-menu#6]
I get this error during startup:

![radial](https://user-images.githubusercontent.com/10487018/31911068-c218ab02-b83f-11e7-912d-3e57dde17fa3.PNG)

I installed it normally using npm.
Deleting node_modules folder, clearing cache and reinstalling did not solve the issue.
It can't find the radial menu module. Other modules like navigation work.

**Comment by omulet** [comment]
@Dantalion this is something related to your packager and the way it resolves project dependencies. 
What is the output if you run this from your project root? `ls node_modules/react-native-radial-menu`

**Comment by Dantalion** [comment]
@omulet Thank you very much for your answer, it was indeed a problem with my packager. 
So I did a fresh setup on a Linux system (In contrary to what the react-native getting started guide recommends I used the latest node, latest npm, latest Android SDK this time) and Android Emulator, now it kind of works. 
Just did a small test with the usage example.
Somehow though it is cutting off the radial menu and pressing it in a small rectangle view box.
But I think this is related to specific styling, etc. 
I'll try to figure out why it did not work at all during the next days, so maybe others with the same problem can fix it easier and faster.
